### ISSUE TITLE
getview vs addview

### ISSUE THREAD

**Issue by danyaljj** [CogComp/cogcomp-nlp#575]
@mssammon  In `Annotator`, getView and addView functions seem to be doing the same thing (one lazy and one not-lazy). Is that really the difference, or anything else I'm missing?

**Comment by mssammon** [comment]
you are correct. 'addView()' is intended to wrap the actual logic for the annotator behavior, while 'getView()' is the public interface. I wanted 'addView()' to be abstract in the base class.

**Comment by danyaljj** [comment]
I think the naming is a little confusing. I wonder if we can merge them together as one name? I don't think it is clear what is the difference between these two functions without looking inside the code.
